automatic/autohotkey.portable/tools/chocolateyInstall.ps1

```diff
@@ -1,6 +1,20 @@
-﻿$packageName = 'autohotkey.portable'
-$url = '{{DownloadUrl}}'
-$unzipLocation = $(Split-Path -parent $MyInvocation.MyCommand.Definition)
-$url64 = '{{DownloadUrlx64}}'
+$ErrorActionPreference = 'Stop'
 
-Install-ChocolateyZipPackage $packageName $url $unzipLocation $url64
+$packageName = 'autohotkey.portable'
+$url32       = 'http://ahkscript.org/download/1.1/AutoHotkey112401.zip'
+$url64       = 'http://ahkscript.org/download/1.1/AutoHotkey112401_x64.zip'
+$checksum32  = '90ECF74C0BDDD243D21B57DB3864EFC3C959CDA7D64296B0A29AD4AC9847E46A'
+$checksum64  = '6E61A6CBBADBB37A114C90E885996DE3A392D6E79AF9D9DF37ACF2DCAA8E39E4'

```

Comment:
So what you see with automatic packaging - you can't simply put the checksums and links in here. You may want to have a look over https://chocolatey.org/docs/automatic-packages first